Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeRotationDecomposeNode.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// include the required headers
+#include "EMotionFXConfig.h"
+#include "AnimGraphNode.h"
+
+
+namespace EMotionFX
+{
+    /**
+     *
+     */
```

Comment:
Maybe add a generic comment here, or remove this?

Reply:
Removed